2 data analysis francis

joelostblom opened the pull request:
---

OrdiNeu commented:
Image broken: img/split-apply-combine.png

Other than that, I'm a bit concerned that we've cut out all of the intro-to-python. Is the PR for lesson 1 going to include all of the into-to-python content that got cut?

Thanks,
Francis Nguyen

joelostblom commented:
Thanks for catching that! Yes, I moved all the intro to Python to lecture
one and removed some repetition from the spreadsheet section. I like that
they get more hands on data frame practice already on the first day.

On Wed, Jul 11, 2018 at 10:06 AM, OrdiNeu <notifications@github.com> wrote:

> Image broken: img/split-apply-combine.png
>
> Other than that, I'm a bit concerned that we've cut out all of the
> intro-to-python. Is the PR for lesson 1 going to include all of the
> into-to-python content that got cut?
>
> Thanks,
> Francis Nguyen
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/UofTCoders/2018-07-12-utoronto/pull/1#issuecomment-404182335>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEWUuRINW3JANhFoTUcOD3BSe0D9Nm__ks5uFgZ0gaJpZM4VKSgp>
> .
>
